nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/pom.xml

```diff
@@ -48,6 +48,16 @@
         </dependency>
         <dependency>
             <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-record-serialization-service-api</artifactId>
+            <scope>compile</scope>
```

Comment:
I think you can leave the scope off the record dependencies because you don't want them actually included in the Mongo NAR... at runtime the Mongo NAR will have a NAR dependency on standard services NAR which is where the record dependencies will live. 

There is a dependencyManagement block in nifi-nar-bundles pom that sets the scope to provided, so that may be overriding the scope here anyway, but just wanted to double check that the record JARs aren't getting bundled into multiple NARs.